core/src/test/scala/integration/kafka/tools/MirrorMakerIntegrationTest.scala

```diff
@@ -35,6 +35,9 @@ import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.Assertions._
 import org.junit.jupiter.api.BeforeEach
 
+import scala.annotation.nowarn
+
+@nowarn("cat=deprecation")
```

Comment:
Can we deprecate the test too instead of the warning suppression?

Reply:
good call, done!